sklearn/multiclass.py

```diff
@@ -915,7 +1068,7 @@ class OutputCodeClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     classes_ : ndarray of shape (n_classes,)
         Array containing labels.
 
-    code_book_ : ndarray of shape (n_classes, code_size)
+    code_book_ : ndarray of shape (n_classes, len(estimators_))
```

Comment:
We have to check in the rendered doc if sphinx try to create an hyperlink with the finishing `_`. In which case we need to put it in backticks.

Reply:
Yes, I had to put it in backticks.